### PR TITLE
Protection against missing discriminators for offline taus in L1 DQM

### PR DESCRIPTION
**Pull request by lathomas**
#### PR description:
This (hopefully) is a short term bug fix of the issue described in https://github.com/cms-sw/cmssw/issues/35304 
In very rare cases, the anti electron discriminators seem to be missing for offline tau, leading to a crash in the L1 DQM code. 
A protection is added to skip such offline taus. 

#### PR validation:

Checked that the crash in https://github.com/cms-sw/cmssw/issues/35304 is gone after this fix. 

@tvami

### CODE REVIEW

**Comment by tvami** [comment]
type bug-fix

**Comment by tvami** [comment]
@cmsbuild , please test

File: DQMOffline/L1Trigger/src/L1TTauOffline.cc [hunk 1]
```diff
@@ -704,6 +704,19 @@ void L1TTauOffline::getProbeTaus(const edm::Event& iEvent,
       TLorentzVector mytau;
       mytau.SetPtEtaPhiE(tauIt->pt(), tauIt->eta(), tauIt->phi(), tauIt->energy());
 
+      if ((*antimu)[tauCandidate].workingPoints.empty()) {
+        LogWarning("This offline tau has no antimu discriminator, skipping");
```
Comment (tvami) [inline DQMOffline/L1Trigger/src/L1TTauOffline.cc:708]: @lathomas isnt the structure with the `LogWarning` something like
```edm::LogWarning  ("category") << "message to be printed"; ```

Comment (tvami) [reply DQMOffline/L1Trigger/src/L1TTauOffline.cc:708]: Yes I think this should be
```edm::LogWarning("L1TTauOffline") << "This offline tau has no antimu discriminator, skipping";```
Please do the same with the other ones too

Comment (lathomas) [reply DQMOffline/L1Trigger/src/L1TTauOffline.cc:708]: Ah yes, will do.

Comment (tvami) [reply DQMOffline/L1Trigger/src/L1TTauOffline.cc:708]: Hi @lathomas please dont forget to push your fix :)

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by lathomas** [comment]
@tvami

**Comment by tvami** [comment]
All looks good, @jfernan2 @rekovic please sign this as soon as possible, so that we can submit the backport and cut a new release for CRAFT, thanks!

**Comment by jfernan2** [comment]
+1

**Comment by tvami** [comment]
Hi @perrotta @qliphy give the urgency and the simplicity of this PR, would you please consider to merge this without the L1 signature?

**Comment by qliphy** [comment]
+1

**Comment by qliphy** [comment]
merge

**Comment by lathomas** [comment]
@tvami I won't be able to take care of the backport(s). Can we identify someone else?

**Comment by tvami** [comment]
> @tvami I won't be able to take care of the backport(s). Can we identify someone else?

I did it myself, please see https://github.com/cms-sw/cmssw/pull/35322

**Pull request merged**